Julia 0.6.1: make install is trying to build documentation again

Hi guys,

I am trying to pack julia 0.6.1 to openSUSE. However, in the build process (`make install`), I am seeing this error:

```
[  497s] make[2]: Entering directory '/home/abuild/rpmbuild/BUILD/julia'
[  497s] Warning: git information unavailable; versioning information limited
[  497s] make[2]: '/home/abuild/rpmbuild/BUILD/julia/usr/lib/julia/sys.so' is up to date.
[  497s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/julia'
[  497s] make[2]: Entering directory '/home/abuild/rpmbuild/BUILD/julia/doc'
[  497s] Warning: git information unavailable; versioning information limited
[  497s] /home/abuild/rpmbuild/BUILD/julia/deps/tools/jldownload http://www.unicode.org/Public/9.0.0/ucd/UnicodeData.txt
[  497s] Could not find working curl, wget, or fetch.
[  497s] You need to install one of these to download dependencies.
[  497s] make[2]: *** [Makefile:22: UnicodeData.txt] Error 1
[  497s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/julia/doc'
[  497s] make[1]: *** [Makefile:119: docs] Error 2
[  497s] make[1]: Leaving directory '/home/abuild/rpmbuild/BUILD/julia'
[  497s] make: *** [Makefile:59: /home/abuild/rpmbuild/BUILD/julia/doc/_build/html/en/index.html] Error 2
[  497s] error: Bad exit status from /var/tmp/rpm-tmp.DCcxzr (%install)
[  497s] 
[  497s] 
[  497s] RPM build errors:
[  497s]     Bad exit status from /var/tmp/rpm-tmp.DCcxzr (%install)
```

This is really strange, because the file `/home/abuild/rpmbuild/BUILD/julia/doc/_build/html/en/index.html` exists. Furthermore, I had no problem with julia v0.6.0 and nothing seems to have changed between v0.6.0 and v0.6.1 with respect to the Makefiles.

--- a/Makefile
+++ b/Makefile
@@ -53,7 +53,7 @@ julia_flisp.boot.inc.phony: julia-deps
 	@$(MAKE) $(QUIET_MAKE) -C $(BUILDROOT)/src julia_flisp.boot.inc.phony
 
 # Build the HTML docs (skipped if already exists, notably in tarballs)
-$(BUILDROOT)/doc/_build/html/en/index.html: $(shell find $(BUILDROOT)/base $(BUILDROOT)/doc \( -path $(BUILDROOT)/doc/_build -o -path $(BUILDROOT)/doc/deps -o -name *_constants.jl -o -name *_h.jl \) -prune -o -type f -print)
+$(BUILDROOT)/doc/_build/html/en/index.html: $(shell find $(BUILDROOT)/base $(BUILDROOT)/doc \( -path $(BUILDROOT)/doc/_build -o -path $(BUILDROOT)/doc/deps -o -name *_constants.jl -o -name *_h.jl -o -name version_git.jl \) -prune -o -type f -print)
 	@$(MAKE) docs
 
 # doc needs to live under $(build_docdir), not under $(build_datarootdir)/julia/

--- a/Makefile
+++ b/Makefile
@@ -146,11 +146,12 @@ release-candidate: release testall
 	@echo 5. Create tag, push to github "\(git tag v\`cat VERSION\` && git push --tags\)"		#"` # These comments deal with incompetent syntax highlighting rules
 	@echo 6. Clean out old .tar.gz files living in deps/, "\`git clean -fdx\`" seems to work	#"`
 	@echo 7. Replace github release tarball with tarballs created from make light-source-dist and make full-source-dist
-	@echo 8. Follow packaging instructions in DISTRIBUTING.md to create binary packages for all platforms
-	@echo 9. Upload to AWS, update https://julialang.org/downloads and http://status.julialang.org/stable links
-	@echo 10. Update checksums on AWS for tarball and packaged binaries
-	@echo 11. Announce on mailing lists
-	@echo 12. Change master to release-0.X in base/version.jl and base/version_git.sh as in 4cb1e20
+	@echo 8. Check that 'make && make install && make test' succeed with unpacked tarballs even without Internet access.
+	@echo 9. Follow packaging instructions in DISTRIBUTING.md to create binary packages for all platforms
+	@echo 10. Upload to AWS, update https://julialang.org/downloads and http://status.julialang.org/stable links
+	@echo 11. Update checksums on AWS for tarball and packaged binaries
+	@echo 12. Announce on mailing lists
+	@echo 13. Change master to release-0.X in base/version.jl and base/version_git.sh as in 4cb1e20
 	@echo
 
 $(build_man1dir)/julia.1: $(JULIAHOME)/doc/man/julia.1 | $(build_man1dir)

